include/deal.II/grid/grid_in.h

```diff
@@ -366,8 +366,13 @@ class GridIn
 
   /**
    * Read grid data from an ucd file. Numerical data is ignored.
+   * It is not possible to use a ucd file to set both boundary_id and
+   * manifold_id for the same cell. Yet it is possible to use
+   * the flag apply_all_indicators_to_manifolds to decide if
+   * the indicators in the file refer to manifolds (flag set to true)
+   * or boundaries (flag set to false).
    */
-  void read_ucd (std::istream &in);
+  void read_ucd (std::istream &in, const bool apply_all_indicators_to_manifolds=false);
```

Comment:
can you break the line so that every argument is on its own line, as we typically do?